intro_to_R_tidyverse/02-intro_to_tidyverse.Rmd

```diff
@@ -219,14 +280,73 @@ Let's use `head(metadata_edit)` so we can look at a preview of what we made:
 head(metadata_edit)
 ```
 
-Let's write this data to a file using `write_tsv`. Note that the second argument 
-needs to be a character string that ends in the appropriate file extension ".tsv"
-and it is what your file you are creating will be named. 
+## Save data to files
+
+#### Save to TSV files
+
+Let's write this data we edited to a file.
+To do this, we can use the `readr` library of `write` functions. 
+Note that the second argument of `write_tsv` needs to be our a character string 
+that contains our `file.path` to the new file we would like to create.
+Remember that we created a `results` directory to put our output in. 
+But if we want to save our data to a directory other than our working directory, 
+we need to specify this. 
+This is what we will use the `file.path` function for. 
+Let's look at what `file.path` does, by taking a look at the print out of it 
+from examples below.
 
 ```{r}
-write_tsv(metadata_edit, "metadata_clean.tsv")
+# Which of these file paths is what we want to use to save our data to the
+# results directory we created at the beginning of this notebook?
+file.path("docker-install", "metadata.tsv")
+file.path("results", "metadata_clean.tsv")
+file.path("metadata_clean.tsv", "results")
 ```
 
+Replace `<NEW_FILE_PATH>` below with the `file.path` from above that will 
+successfully save our file to the `results` folder 
+
+```{r eval=FALSE}
+# Write our data.frame to a TSV file
+write_tsv(metadata_edit, <NEW_FILE_PATH>)
+```
+
+Check in your `results` directory to see if your new file has successfully saved.
+
+#### Save to RDS files
+
+For this example we have been working with `data.frame`s, however, in other 
+situations we may want to save more complicated structure objects or very large 
+objects. 
+RDS files may be a better option in these instances.
+RDS is R's special file format for holding data exactly as you have it in your 
+R environment. 
+RDS files also are compressed, meaning they will take up less space on your 
```

Comment:
```suggestion
RDS files can also be compressed, meaning they will take up less space on your 
```
`write_rds` does not compress by default, the base function `saveRDS` does. I have typically used `saveRDS` outside of the training context so I probably misspoke a bit last training.

Reply:
Ah I see. Yeah, I usually use `saveRDS` so I didn't even think about it.